### ISSUE TITLE
Docs

### ISSUE THREAD

**Issue by Tropix126** [rgossiaux/svelte-headlessui#23]
Aware that this is already planned by the readme, but I thought that I might open this issue to track it anyways. Might suggest a few things, since documenting svelte components can be made a lot easier than expected.

Few neat things that can help:
- https://mdsvex.pngwn.io/ is useful for embedding components in markdown (svx files can also serve as SvelteKit routes).
- https://github.com/mattjennings/vite-plugin-sveld automatically generates component API documentation based on typescript definitions and JSDoc comments in your components.

**Comment by rgossiaux** [comment]
Thanks for filing & for the tooling suggestions! I definitely do intend on including project-specific docs here. 

At the moment I think the https://headlessui.dev React docs hopefully work well enough to be usable & so I've been focusing on porting over the test files. After that's finished I'll turn back to the docs.

**Comment by jeffpohlmeyer** [comment]
> Thanks for filing & for the tooling suggestions! I definitely do intend on including project-specific docs here.
> 
> At the moment I think the https://headlessui.dev React docs hopefully work well enough to be usable & so I've been focusing on porting over the test files. After that's finished I'll turn back to the docs.

Are all of the components listed there available here as of now or are there some that are still to be ported over?

**Comment by rgossiaux** [comment]
Yep, they're all available! It's a full port :)

**Comment by jeffpohlmeyer** [comment]
Fantastic news. A couple of months ago I was thinking of doing this myself so it's definitely nice to see. Thanks for doing this

**Comment by cogoo** [comment]
> Fantastic news. A couple of months ago I was thinking of doing this myself so it's definitely nice to see. Thanks for doing this

Exactly the same thought!

Thank you so much @rgossiaux for this!

I suppose we can piggy back of the React docs, but just show how to transform into the svelte version?

**Comment by vhscom** [comment]
I started porting the [official Tailwind UI components](https://tailwindui.com/preview) which were given away free by Tailwind Labs here: https://codeberg.org/vhs/svelte-tailwindui-freebies.

Having the freebies ported to Svelte would serve as living documentation as well as a test scaffold for this library. I've already ported the navigation and slide-out examples. If anyone wants to help porting the others, open the `$lib` folder and pull up the official component docs for React/Vue and have at it.
